### DISCUSSION TITLE
Manual IP address

### DISCUSSION THREAD

**Discussion by Will-Bee** [elmoiv/elmocut#52] [Ideas]
Sometimes, elmocut just dont see all of the hosts in network, but for example nmap is capable of it. So, could you add some option for manually adding tergets?

**Comment by elmoiv** [comment]
Can you try to set device count and threads count to maximum and try again:

![image](https://user-images.githubusercontent.com/31712173/222111559-a18e658f-7ebe-4023-be2a-4eb084b95131.png)
